### DISCUSSION TITLE
IPv6 supported?

### DISCUSSION THREAD

**Discussion by Fasibru** [Azure/ipam#260] [General]
Hi @DCMattyG,
I'm wondering if IPv6 is supported by the tool? I took a look at the codebase but everything that is related to ipv6 seems to be commented out and the docs didn't reveal an answer either.

**Comment by DCMattyG** [comment]
Hi there @Fasibru, you are correct that there are some components of IPv6 in the code that are currently commented out. That is because while the tool does not support IPv6 today, some of the framework to add that in has already been added, but we want to make sure that IPv6 is supported end-to-end in the tool and not just in certain components.

IPv6 support is on the roadmap, but not quite there today. If we were to start enabling IPv6, what are some of the key features that you would prefer enabled immediately (priority) assuming we weren't ready to enable it end-to-end quite yet?

**Reply by Fasibru** [reply]
Thanks for the confirmation. It's difficult to assign priorities because for me it would be important that ipv6 is supported from spaces and blocks down to vnets and subnets including reservations with full API support for all actions - and I guess that's already a big chunk of the end-to-end workflow. I'm not sure if it makes sense to role out support for ipv6 only for parts of the workflow but maybe I'm missing some aspects here.
